### PR TITLE
Test count star for hive storage formats

### PR DESCRIPTION
**Pull request by kokosing**
Test count star for hive storage formats

### CODE REVIEW

**Comment by findepi** [comment]
> this test is really about “select no columns”.

I think `count(*)` is a special case, because we read files, but only the metadata portion of them.

“select no columns” does not convey this 

Can we have something that is less SQL-ish but still conveys the meaning?

**Comment by findepi** [comment]
`testCount`

**Comment by kokosing** [comment]
Theoretically `count(*)` might be using different execution path than `SELECT 1 FROM x`. However, my point to here was to add a `select no columns` test. Anyway, I will also add a test for `SELECT 1 FROM x`.

**Comment by kokosing** [comment]
Updated

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java [hunk 1]
```diff
@@ -3816,6 +3816,28 @@ private HiveInsertTableHandle getHiveInsertTableHandle(Session session, String t
...
+    public void testSelectWithNoColumns()
+    {
+        testWithAllStorageFormats(this::testSelectWithNoColumns);
+    }
+
+    private void testSelectWithNoColumns(Session session, HiveStorageFormat storageFormat)
+    {
+        String tableName = "test_select_with_no_columns";
+        @Language("SQL") String createTable = format(
+                "CREATE TABLE %s (col0) WITH (format = '%s') AS VALUES 5,6,7",
```
Comment (findepi) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java]:
```suggestion
                "CREATE TABLE %s (col0) WITH (format = '%s') AS VALUES 5, 6, 7",
```

File: presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java [hunk 2]
```diff
@@ -3816,6 +3816,28 @@ private HiveInsertTableHandle getHiveInsertTableHandle(Session session, String t
...
+                tableName,
+                storageFormat);
+        assertUpdate(session, createTable, 3);
+        assertTrue(getQueryRunner().tableExists(getSession(), tableName));
+
+        assertQuery("SELECT 1 FROM " + tableName, "VALUES 1,1,1");
```
Comment (findepi) [inline presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java]:
```suggestion
        assertQuery("SELECT 1 FROM " + tableName, "VALUES 1, 1, 1");
```

**Pull request merged**